caravel/migrations/versions/b922b43a4a2c_change_table_unique_constraint.py

```diff
@@ -0,0 +1,74 @@
+"""change_table_unique_constraint
+
+Revision ID: b922b43a4a2c
+Revises: f162a1dea4c4
+Create Date: 2016-07-20 22:16:05.087720
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'b922b43a4a2c'
+down_revision = 'f162a1dea4c4'
+
+from alembic import op
+from caravel import db
+from caravel.utils import generic_find_unique_constraint_name
+import logging
+
+naming_convention = {
+    "uq": "uq_%(table_name)s_%(column_0_name)s"
+}
+
+
+def default_constraint_name(table_name, column_name):
+    return naming_convention['uq'] % {'table_name': table_name,
+                                      'column_0_name': column_name}
+
+
+def find_constraint_name():
+    return generic_find_unique_constraint_name(
+        table='tables', columns=['table_name'], db=db)
```

Comment:
I'd add the `or default_constraint_name(...)` inside find_constraint_name so we don't have too much wrappers